lalrpop/Cargo.toml

```diff
@@ -24,7 +24,7 @@ is-terminal = "0.4.2"
 itertools = { version = "0.10", default_features = false, features = ["use_std"] }
 petgraph = { version = "0.6", default_features = false }
 regex = { version = "1", default_features = false, features = ["std"] }
-regex-syntax = { version = "0.6", default_features = false }
+regex-syntax = { version = "0.6", default_features = false, features = ["unicode"] }
```

Comment:
Rather than making this change, let's pin `regex` to `1.7` for a while.
```
regex = { version = "=1.7", default_features = false, features = ["std"] }
```

Reply:
Why? `=` constraints are really bad for the ecosystem and can lead to unresolveable build failures. And `lalrpop` is compatible with regex `1.8`.

Reply:
Oh, yep. I got something wrong while changing regex-syntax version a few times.